guides/openstack.md

```diff
@@ -92,6 +93,220 @@ There are some differences in terminology between jclouds and OpenStack that sho
   </div>
 </div>
 
+## <a id="keystone"></a>Keystone v2-v3 authentication
+
+Openstack Keystone (aka: [OpenStack Identity Service](https://docs.openstack.org/keystone/latest/)) has major changes between v2 and v3 (detail. [Identity API v2.0 and v3 History](https://docs.openstack.org/keystone/latest/contributor/http-api.html)).
```

Comment:
[minor] Quite a few instances of "Openstack" (lowercase `s`) rather than "OpenStack" here - is that worth cleaning up?

Reply:
In [official documentation](https://docs.openstack.org/latest/), "S" is in uppercase. I can do the cleanup in a way or the other, but we should choose :)